docs/start/integration.md

```diff
@@ -5,111 +5,117 @@ title: Integration Guide
 
 :::info
 
-We're currently working on the new version of the API. It's currently available as `v2alpha` but we strongly recommend doing any new integrations with it. The old version `v1` will be deprecated afterwards. The API description is available [here](https://github.com/spacemeshos/api/tree/master/spacemesh/v2alpha1).
-There is also public instance exposing the JSON-RPC api at `mainnet-api.spacemesh.network`
+We are currently working on the new version of the Spacemesh API. It is currently available as `v2alpha` but we strongly recommend doing any new integrations with it. The old version `v1` will be deprecated afterwards. The API description is available [here](https://github.com/spacemeshos/api/tree/master/spacemesh/v2alpha1).
```

Comment:
If we're "strongly recommending" that people integrate with the new API, it's time to remove "alpha" from the name. This is confusing.

Reply:
Agree. What should I change here?

Reply:
We'd like to keep `alpha` to iterate quickly and then move to `beta` with 1.7.

Reply:
I disagree, but this isn't the place to debate it